rh-jmc-team/container-jfr
# Bump web-client to v0.14.0

To test:
```
mvn clean verify && podman image prune -f && unset CONTAINER_JFR_CORS_ORIGIN && CONTAINER_JFR_RJMX_USER=someuser CONTAINER_JFR_RJMX_PASS=somepass sh run.sh
```

Then open `http://0.0.0.0:8181` in a browser and verify that things work as expected.
